features/google/src/views/explore/explore.css

```diff
@@ -0,0 +1,94 @@
+.explore-view {
+    background-color: var(--background-dark);
+    color: var(--color-gray-50);
+    line-height: var(--line-height);
+    letter-spacing: -0.01em;
```

Comment:
This `letter-spacing` thing gives me a few headaches. For starters, it's not consistent; it's either 0.01 or 0.02. Also, it's everywhere:

```
feature-utils/poly-look/src/css/themes.css:  letter-spacing: -0.01em;
features/facebookImport/src/components/dataViz/barChart.css:    letter-spacing: -0.02em;
features/facebookImport/src/components/importExplanationExpandable/importExplanationExpandable.css:    letter-spacing: -0.01em;
features/facebookImport/src/components/importExplanationExpandable/importExplanationExpandable.css:    letter-spacing: -0.01em;
features/facebookImport/src/components/infoBox/infoBox.css:    letter-spacing: -0.01em;
features/facebookImport/src/components/onOffFacebookMiniStory/onOffFacebookMiniStory.css:    letter-spacing: -0.01em;
features/facebookImport/src/model/analyses/ministories/ministories.css:    letter-spacing: -0.01em;
features/facebookImport/src/styles.css:    letter-spacing: -0.01em;
features/facebookImport/src/views/explore/explore.css:    letter-spacing: -0.01em;
features/facebookImport/src/views/overview/overview.css:    letter-spacing: -0.01em;
features/facebookImport/src/views/report/report.css:    letter-spacing: -0.01em;
features/polyExplorer/src/components/clusterStories/sectionTitle.css:    letter-spacing: -0.01em;
features/polyExplorer/src/components/clusterStories/sectionTitle.css:    letter-spacing: -0.01em;
features/polyExplorer/src/components/clusterStory/clusterStory.css:    letter-spacing: -0.01em;
features/polyExplorer/src/components/clusterStory/clusterStory.css:    letter-spacing: -0.01em;
features/polyExplorer/src/components/dataViz/purposeChart.css:    letter-spacing: -0.02em;
features/polyExplorer/src/components/sectionTitle/sectionTitle.css:    letter-spacing: -0.01em;
features/polyPreview/src/styles.css:    letter-spacing: -0.01em;
```

(also in SVG files). So two questions
1. Are you sure this is what you're looking for, as in supported by the Figma design?
2. Are you sure this is not already defined higher up the hierarchy? I see it's defined in `themes`, for instance.
3. Even if that's not the case, and since it's all over, wouldn't it be better to define a variable and defin it centrally?

Reply:
yeah, that's a good point, I just copied the explore css in facebook to have some template styling in place. Not really any design considerations went into this, just used it as place to start from :). Getting it from themes or a variable as you've suggested would make sense :)